r2/r2/templates/base.html

```diff
@@ -30,7 +30,8 @@
   <head>
     <title>${self.Title()}</title>
 
-    <meta name="viewport" content="initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0"/>
+<!-- `meta name="viewport"` set for mobile browsers. See https://github.com/tog22/eaforum/issues/7 & https://github.com/tog22/eaforum/issues/28 -->
```

Comment:
It's also worth noting that that comment will be downloaded and rendered in HTML every single time the base template is rendered (which if memory serves is pretty much every page on the site).